tests/virt_autotest/transactional/prepare_transactional_server.pm

```diff
@@ -0,0 +1,117 @@
+# PREPARE TRANSACTIONAL SERVER MODULE
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This module do preparation work for virtual machine installation on
+# transactional server, including extensions registration, packages installation,
+# services toggle and other operations that are pertinent to transactonal server
+# only or which need to to performed by leveraging transactional-update command.
+#
+# Maintainer: Wayne Chen <wchen@suse.com>
```

Comment:
Would you please also add qe-virt ML into maintainer?

Reply:
Done